hpcgap/lib/info.gi

```diff
@@ -172,24 +173,23 @@ InstallGlobalFunction( DeclareInfoClass, function( name )
     fi;
 end );
 
-#F  SetInfoOutput( <class>, <handler> )
+#############################################################################
 ##
-InstallGlobalFunction( SetInfoOutput, function(class, out)
+#F  SetInfoHandler( <class>, <handler> )
+##
+InstallGlobalFunction( SetInfoHandler, function(class, handler)
     atomic readwrite InfoData do
-    InfoData.Output[class![1]] := out;
-od;
-
+      InfoData.Handler[class![1]] := handler;
+    od;
 end);
 
 #############################################################################
 ##
-#############################################################################
-##
-#F  SetInfoHandler( <class>, <handler> )
+#F  SetInfoOutput( <class>, <handler> )
 ##
-InstallGlobalFunction( SetInfoHandler, function(class, handler)
+InstallGlobalFunction( SetInfoOutput, function(class, out)
     atomic readwrite InfoData do
-      InfoData.Handler[class![1]] := handler;
+    InfoData.Output[class![1]] := out;
```

Comment:
Hmm, yeah, I guess; what happened is that I noticed the two `InstallGlobalFunction` calls were switched on the HPC-GAP side, and the comments obviously messed up (see e.g. lines 184+185 in the old version of the code. So I just blindly switched them; didn't touch the formatting at all.